src/components/SummaryWeather.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useWeather } from './WeatherProvider';
+import {
+  toCapitalized,
+  toLocalDate,
+  getHourString,
+  getIconSrc,
+} from '../utilities/helperFunctions';
+
+export default function SummaryWeather() {
+  const { currentWeather, hourlyWeather } = useWeather();
+  if (!(currentWeather && hourlyWeather)) return null;
+
+  const { timezone } = currentWeather;
+  const summaryHeader = `${currentWeather.name}, ${currentWeather.sys.country}`;
+  const summaryIconURL = getIconSrc(currentWeather.weather[0].icon);
+  const summaryIconAltText = `An icon representing ${currentWeather.weather[0].description}`;
+  const summaryDescription = `${toCapitalized(
+    currentWeather.weather[0].description
+  )}`;
+  const summaryTemperature = `${Math.round(currentWeather.main.temp)}°`;
+
+  const hourlyForecast = hourlyWeather.list
+    .slice(0, 4)
+    .map((weatherForHour) => (
+      <div key={weatherForHour.dt} className="row align-items-center">
+        <div className="col" />
+        <div className="col fs-4">
+          {getHourString(toLocalDate(weatherForHour.dt), timezone)}
+        </div>
+        <div className="col">
+          <img
+            src={getIconSrc(weatherForHour.weather[0].icon)}
+            alt={`An icon representing ${weatherForHour.weather[0].description}`}
+          />
+        </div>
+        <div className="col" />
+      </div>
+    ));
+
+  return (
+    <div className="summary-weather container py-4 text-center border border-primary rounded">
+      <h2 className="row justify-content-center fs-1">{summaryHeader}</h2>
+      <h3 className="row justify-content-center fs-1">{summaryTemperature}</h3>
+      <img
+        src={summaryIconURL}
+        alt={summaryIconAltText}
+        height="200px"
+        width="200px"
+      />
+      <h3 className="row justify-content-center fs-2 mb-2">
+        {summaryDescription}
+      </h3>
+      {hourlyForecast}
+    </div>
+  );
+}
```

Comment:
After taking a look at the sidebar, although this would require a bit more work refactoring, using [bootstrap flex](https://getbootstrap.com/docs/5.2/utilities/flex/) instead of grid would be a lot cleaner because the sidebar is one dimensional. With flex, you can declare the parent container using `d-flex` and `flex-column` then each child would automatically be aligned accordingly. There would be no need to use rows, columns, etc.

Reply:
Sorry, I wouldn't want you to refactor all your code changes b/c it does work! Just throwing out some ideas because it may be a bit cleaner. I refactored it a little bit to use bootstrap flex and I'll post my changes in the discord to show you what I did. Feel free to look at it and change it if you'd like.